example/gluon/word_language_model/train.py

```diff
@@ -125,19 +119,19 @@ def train():
             hidden = detach(hidden)
             with autograd.record():
                 output, hidden = model(data, hidden)
-                L = loss(output, target)
+                L = mx.nd.mean(loss(output, target))
```

Comment:
we usually use sum to make multi gpu training easier. Does this affect accuracy?